lightcurve/static/lightcurve/js/lightcurve.js

```diff
@@ -61,4 +73,19 @@ $(function() {
     window.location.reload();
     return false;
   });
+
+  $(document).on('click', '.edit-name', function(e) {
```

Comment:
You should prefer the form `$('.js-edit-name').on('click', function() { ... })`.  Your current approach sets an event listener on the entire document.  Internally, jquery will catch all events and trigger your function for targets matching your selector.  It is simpler (and maybe a little more efficient) to attach the event directly to the button.

Reply:
Done.